ras_backstage/__init__.py

```diff
@@ -27,6 +27,7 @@
 sample_api = Namespace('sample', path='/backstage-api/v1/sample')
 secure_messaging_api = Namespace('secure-messaging', path='/backstage-api/v1/secure-message')
 sign_in_api = Namespace('sign-in', path='/backstage-api/v1/sign-in')
+sign_in_uaa_api = Namespace('sign-in-uaa', path='/backstage-api/v1/sign-in-uaa')
```

Comment:
Do we need to support both?

I'd rather get rid of the old one, as we'll replace it eventually anyway.

If we do then put it as v2?
i.e.
/backstage-api/v2/sign-in-uaa

Reply:
We need to support both right now.
It's being used by the angular application which is still in use for BRES